modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/sql/fun/IgniteSqlOperatorTable.java

```diff
@@ -103,10 +102,16 @@ public class IgniteSqlOperatorTable extends ReflectiveSqlOperatorTable {
      * Generic {@code SUBSTR(string, position [, length]} function.
      * This function works exactly the same as {@link SqlSubstringFunction SUSBSTRING(string, position [, length])}.
      */
-    public static final SqlFunction SUBSTR = new SqlFunction("SUBSTR", SqlKind.OTHER_FUNCTION,
-            ReturnTypes.ARG0_NULLABLE_VARYING, null,
-            OperandTypes.STRING_INTEGER_OPTIONAL_INTEGER,
-            SqlFunctionCategory.STRING);
+    public static final SqlFunction SUBSTRING = new SqlSubstringFunction();
```

Comment:
I think `SUBSTRING` and  `SUBSTR`, since a comment over `SUBSTRING` belongs to `SUBSTR`.